google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/benchmark/README.md

```diff
@@ -0,0 +1,9 @@
+# BigQuery Benchmark
```

Comment:
Is it maybe more appropriate to put this under src/test or some other directory besides src/main?

Reply:
Do you have a directory in mind? I was thinking `src/benchmark` but that felt a little odd since it's not a standard Maven directory.

Another thought: create another project `google-cloud-benchmarks`.

Reply:
It's true that `src/benchmark` is not a standard Maven directory, but it's also not standard to do benchmarks. I think it follows the spirit of Maven though. I'd prefer to keep `google-cloud-*` directories mostly limited to APIs.

Reply:
Done